server/src/main/java/org/elasticsearch/script/ScoreScript.java

```diff
@@ -77,6 +83,11 @@ public void setScorer(Scorable scorer) {
         };
     }
 
+    /** Return the score of the current document. */
+    public double getScore() {
```

Comment:
Why is this added? This would make `score` available in addition to `_score` (which is already available with `get_score()` below). Why do we need the additional score accessor, that deviates from the access in other scripts?